<launch>

    <!-- 
        The topic that we'll send the commands to.
        By default, we want want to directly control the robot
     -->
    <arg name="twist_topic" default="/robot/cmd_vel" />

    <!-- GPS Manager -->
    <include file="$(find decision)/launch/gps_manager.launch" />

    <!-- GPS Decision -->
    <include file="$(find decision)/launch/gps_decision.launch" >
        <arg name="output_topic" value="$(arg twist_topic)" />
    </include>

</launch>
